first-year-parse/first-year-new.csv

```diff
@@ -0,0 +1,40 @@
+CS,CS10001,Programming and Data Structures,--,3-0-0,3,A3,NR122

```

Comment:
I don't understand why are there two files `first-year-new.csv` and `first-year.csv`. Is there any particular reason or is it by mistake?

Reply:
That's just to get metadata about the courses.

Reply:
first-year.csv is the previous one to get metadata about the courses in the new CSV.

Reply:
Is it always needed? We should be mentioning that in the README.md itself.